content/medo_manual/inloggen_en_veiligheid.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Inloggen en veiligheid
+sort: 100
+---
+
+* TOC
+{:toc}
+
+## Eerste gebruik
+
+U ontvangt van ons een e-mail met de activatielink wanneer u uitgenodigd bent
+om deel te nemen aan het beheer van een Medo project. Wanneer u op deze link
+klikt, moet u uw naam invullen en een nieuw wachtwoord kiezen. Hierna wordt
+u doorverwezen naar een scherm waar u een
+[twee-staps-verificatie](#two-factor-authentication) kunt instellen.
+
+## Two-Factor Authentication
```

Comment:
Toch wel handig om alle verwijzingen naar `Two-Factor Authentication` naar `twee-staps-verificatie` te verwijderen, ik betwijfel of mijn moeder direct zou doorhebben dat ze hetzelfde betekenen